forem/forem
# Accessibility: initial onboarding fixes

## What type of PR is this? (check all applicable)


## Description

This PR includes some of the initial accessibility issues I encountered upon setup. It primarily updates color contrast to pass WCAG 2.0 AA but there's also one bit of missing alt text. Additional accessibility items will be addressed in future PRs.

## Related Tickets & Documents

## QA Instructions, Screenshots, Recordings

- Run the [axe extension](https://deque.com/axe) on DEV.to
- See the color contrast issues that come up
- Run axe again on this PR
- Compare it to the list of accessibility violations

## Added tests?

- [X] No, and this is why: Not applicable

## Added to documentation?


## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![Simpsons GIF of Lisa saying 'wow, look at all the colors, man!' and Milhouse saying 'I'm FREAKING OUT!'](https://media.giphy.com/media/z0zTHzcwM4VYQ/giphy.gif)
